src/Models/QueryOptions.cs

```diff
@@ -1,14 +1,14 @@
+using System;
 using System.Collections.Generic;
 using Newtonsoft.Json;
-using System;
-using StreamChat.Rest;
 
-namespace StreamChat
+namespace StreamChat.Models
 {
     public enum SortDirection
     {
+        Unknown,
```

Comment:
It is a good practise to have a default value in `enums`. Since they're value types, they'll always have the default value if not defined otherwise. It' usually `None`, `Unknown` things like that.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/enum?redirectedfrom=MSDN

> ✔️ DO provide a value of zero on simple enums.

> Consider calling the value something like "None." If such a value is not appropriate for this particular enum, the most common default value for the enum should be assigned the underlying value of zero.

Reply:
I'll rename it to `None` actually